ld/K64FN1M0xxx12.ld

```diff
@@ -7,7 +7,7 @@ MEMORY
   VECTORS (rx)          : ORIGIN = 0x00000000, LENGTH = 0x00000400
   FLASH_PROTECTION (rx) : ORIGIN = 0x00000400, LENGTH = 0x00000010
   FLASH (rx)            : ORIGIN = 0x00000410, LENGTH = 0x00100000 - 0x00000410
-  RAM (rwx)             : ORIGIN = 0x1FFF0000, LENGTH = 0x00010000 
+  RAM (rwx)             : ORIGIN = 0x1FFF0000, LENGTH = 0x00020000 
```

Comment:
That doesn't look right. You're overflowing RAM into RAM2 now, what's the point of this?